models/oneinch/fantom/oneinch_fantom_calls_transfers.sql

```diff
@@ -0,0 +1,126 @@
+{{ config( 
+    alias = alias('fantom_calls_transfers'),
+    tags = ['dunesql'],
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['tx_hash', 'start', 'transfer_trace_address']
+    )
+}}
+
+{% set project_start_date = "timestamp '2020-11-13'" %} 
+{% set lookback_days = -2 %}
```

Comment:
our spellbook-wide standard is to look back 7 days. it's mostly as a safeguard for late arriving blocks and fixing bugs. you can open it up a few more days to the standard

Reply:
fair enough, will change to 7